src/pages/portfolio-page/index.tsx

```diff
@@ -153,3 +203,53 @@ const PortfolioPage = () => {
 };
 
 export default PortfolioPage;
+
+export const pageQuery = graphql`
+  query IndexPageQuery {
+    portfolioPage: markdownRemark(fileAbsolutePath: { regex: "/portfolio/index-1.md/" }) {
+      frontmatter {
+        portfolio_page_title
+        projects {
+          project_label
+          project_code
+          project_description
+          project_role
+          project_preview_note
+          project_app
+          project_start_date
+          project_finish_date
+          project_mockups
+          project_preview_image {
+            publicURL
+          }
+          project_technologies {
+            technology_name
+          }
+          project_name
+        }
+      }
+    }
+    developerProfile: markdownRemark(fileAbsolutePath: { regex: "/developer-profile/index-1.md/" }) {
```

Comment:
DetailsCard perform in more than one page component, so maybe you can move this 
query to other query or use useStaticQuery directly in the DetailsCard component and then everybody will use the same query and we won't duplicate our code ?

Reply:
Yeah, I think that it's a great idea, I will try to use it directly in component 👍